cmd/node/main.go

```diff
@@ -259,19 +258,16 @@ func prepareCRDControllerManagers(volumeCtrl *node.CSINodeService, lvgCtrl *lvg.
 	return mgr
 }
 
-func getNodeID(client k8sClient.Client, nodeName string, featureChecker featureconfig.FeatureChecker) (string, error) {
+func getNodeID(client *k8s.KubeClient, nodeName string, featureChecker featureconfig.FeatureChecker) (string, error) {
```

Comment:
Can we use same `getNodeId` method in `node` and `loopbackmgr`?

Reply:
Yes. Is `KubeClient` a good place for this function?